series_items.go

```diff
@@ -34,6 +36,7 @@ func RemoveFromSeries(seriesId string, itemType string, itemId string, time_ flo
 	}
 }
 
+// ListSeriesItems lists all the items present in the given series, sorted according to their time index values.
```

Comment:
also it is possible to list series of series (season) elements. In such case series list returns series of series (seasons)